core/modules/update/config/update.settings.json

```diff
@@ -12,5 +12,11 @@
     "update_system_info": [],
     "update_status": [],
     "update_xml_map": 0,
-    "update_projects": []
+    "update_projects": [],
+    "project_browser_server": {
+        "url": "https://projects.backdropcms.org",
+        "name": "Backdrop",
+        "method": "json"
+        
```

Comment:
As noted above, perhaps just remove "method" entirely and assume JSON. We don't support XMLRPC.